client/my-sites/themes/theme-preview.jsx

```diff
@@ -1,39 +1,26 @@
-import { isEnabled } from '@automattic/calypso-config';
 import { Button } from '@automattic/components';
-import { getDesignPreviewUrl } from '@automattic/design-picker';
-import { createHigherOrderComponent } from '@wordpress/compose';
 import { localize } from 'i18n-calypso';
-import page from 'page';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
 import { connect } from 'react-redux';
 import QueryCanonicalTheme from 'calypso/components/data/query-canonical-theme';
-import PremiumGlobalStylesUpgradeModal from 'calypso/components/premium-global-styles-upgrade-modal';
 import PulsingDot from 'calypso/components/pulsing-dot';
-import ThemePreviewModal from 'calypso/components/theme-preview-modal';
```

Comment:
Should we remove this component from the code base as well? 🤔 We might need it in the future.

Reply:
Same